Dockerfile

```
@@ -1,9 +1,10 @@
-FROM node:14
+FROM node:18 as base
+
+FROM base as deps
```

Maybe it's a convention? But is there a reason that we immediately change to a different named build step without doing anything with base?
It seems like we could condense these first two lines to:
```suggestion
FROM node:18 as deps
```
because we're not using base anywhere else (we're using node:18-slim instead to run the app below).